helpers/deploymentConfig.ts

```diff
@@ -1015,6 +1025,9 @@ export const assets: Assets = {
       oracle: "redstone",
     },
   ],
+  opsepolia: [
+    // TODO: add tokens
+  ],
```

Comment:
We need to deploy first the mock tokens in the IL repo

Reply:
Will update the addresses here once tokens are deployed.